package.json

```diff
@@ -177,7 +177,6 @@
     "balena-semver": "^2.2.0",
     "balena-settings-client": "^4.0.4",
     "balena-sync": "^10.2.0",
-    "bash": "0.0.1",
```

Comment:
It's been version 0.0.1 for 9 years, :-) and just over 1K downloads a week. Anyway, no longer needed. The CLI also uses another dependency that overlaps in functionality: `shell-escape`.